test/fixtures/projects/android.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+
+const manifest = fs.readFileSync(path.join(__dirname, '../files/AndroidManifest.xml'));
+const mainJavaClass = fs.readFileSync(path.join(__dirname, '../files/Main.java'));
+
+exports.valid = {
+  src: {
+    'AndroidManifest.xml': manifest,
+    main: {
```

Comment:
With declaration: `main/com/some/example: {}` glob does not work, so needs to be declared as a tree so every folder is mocked.